cloud/scope/client.go

```diff
@@ -7,19 +7,55 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// LinodeObjectStorageClient defines functions suitable for provisioning object storage buckets and keys.
+// LinodeClient is an interface that defines the methods that a Linode client must have to interact with Linode.
+// It defines all the functions that are required to create, delete, and get resources
+// from Linode such as object storage buckets, node balancers, linodes, and VPCs.
+type MachineScopeLinodeClient interface {
+	LinodeNodeBalancerClient
+	LinodeInstanceClient
+	LinodeVPCClient
+}
```

Comment:
Could be good to get some feedback from others on what to name this interface. Having `Machine` in the name definitely feels appropriate. Maybe not include the `Scope` part?

Reply:
Yup def a good idea. It was tough naming this haha